db/apps/BetMas/expath-pkg.xml

```diff
@@ -1,5 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <package xmlns="http://expath.org/ns/pkg" name="https://www.betamasaheft.uni-hamburg.de/BetMas" abbrev="BetaMasaheft" version="5.0" spec="1.0">
     <title>Beta maṣāḥǝft: Manuscripts of Ethiopia and Eritrea</title>
-    <dependency package="http://exist-db.org/apps/shared"/>
+    <dependency package="http://exist-db.org/apps/shared" semver-min="0.9.1"/>
+    <dependency package="https://betamasaheft.eu/betmasweb/" semver-min="0.1"/>
```

Comment:
I wonder why BetMas - as the legacy application - depends on its successor betmasweb?
Will this not introduce a circular dependency, when the hidden dependency in betmasweb on betmas is declared?

Reply:
yes it will, so before we actually deploy anything we need to resolve these properly. But my goal for now is to declare the current status, warts n all .